msp/identities.go

```diff
@@ -181,8 +181,9 @@ func (id *identity) Verify(msg []byte, sig []byte) error {
 	}
 
 	if mspIdentityLogger.IsEnabledFor(zapcore.DebugLevel) {
-		mspIdentityLogger.Debugf("Verify: digest = %s", hex.Dump(digest))
-		mspIdentityLogger.Debugf("Verify: sig = %s", hex.Dump(sig))
+		mspIdentityLogger.Debugf("Verify: signer identity (certificate subject=%s issuer=%s serialnumber=%d)", id.cert.Subject, id.cert.Issuer, id.cert.SerialNumber)
+		//mspIdentityLogger.Debugf("Verify: digest = %s", hex.Dump(digest))
+		//mspIdentityLogger.Debugf("Verify: sig = %s", hex.Dump(sig))
```

Comment:
Discussed with the crypto experts in the main branch PR and we decided to leave them as commented out so that it is easy to re-enable in some environment if needed.